geoprocessing/src/migrations/geoprocessing/1621847467456-ScenariosPuCostDataId.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ScenariosPuCostDataId1621847467456 implements MigrationInterface {
+  name = 'ScenariosPuCostDataId1621847467456';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "scenarios_pu_cost_data" ADD "scenarios_pu_data_id" uuid`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "scenarios_pu_cost_data" ADD CONSTRAINT "FK_21454fad6e954ba771262974ae7" FOREIGN KEY ("scenarios_pu_data_id") REFERENCES "scenarios_pu_data"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`,
```

Comment:
I imagine you have discussed this with Alicia - just cross checking that the absence of cascades here is intentional. I can't think of a reason to leave cost data behind in case we delete scenario PUs

Reply:
actually I missed it - thanks for catching this out!
fixed in https://github.com/Vizzuality/marxan-cloud/pull/206/commits/af3773cc52442b0444333f23e50e9fdd5f43580e